robot_skills/src/robot_skills/perception.py

```diff
@@ -267,7 +271,7 @@ def get_face_properties(self, faces=None, image=None):
             face_properties_response = self._face_properties_srv(faces)
             face_properties = face_properties_response.properties_array
         except Exception as e:
-            rospy.logerr(str(e))
+            rospy.logerr(e)
```

Comment:
You're sure this works and doesn't expect a string?

Reply:
Yep, Python resolves this automatically to the string representation of the Exception.